fofix/game/SongChoosingScene.py

```diff
@@ -655,7 +656,8 @@ def checkParts(self):
         return True
 
     def checkQueueParts(self):
-        parts = self.engine.world.songQueue.getParts()
+        #TODO: checkQueueParts
+        pass #parts = self.engine.world.songQueue.getParts()
```

Comment:
This method seems not to be used. Please, put `pass` after the commented line ;).